tests/reduction.py

```diff
@@ -1,145 +0,0 @@
-# Copyright 2021-2022 NVIDIA Corporation
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-#
-
-import numpy as np
-from test_tools import asserts
-
-import cunumeric as num
-
-
-def test():
-    x = num.array([])
-    r = num.sum(x)
-    assert r == 0
-
-    x = num.array([1])
-    r = num.sum(x)
-
-    assert r == 1
-
-    x = num.eye(3)
-    r = num.sum(x)
-
-    assert r == 3
-
-    x = num.array([1, 2, 3, 4.0])
-    r = num.sum(x)
-    r2 = num.add.reduce(x)
-
-    assert r == r2 == 10
-
-    x = num.array([1, 2, 3, 4.0, 5.0])
-    r = num.prod(x)
-    r2 = num.multiply.reduce(x)
-    assert r == r2 == 120
-
-    asserts.assert_equal(num.sum([]), np.sum([]))
-    asserts.assert_equal(num.add.reduce([]), np.add.reduce([]))
```

Comment:
I actually prefer removing individual lines using `ufunc.reduce`, instead of removing this test completely.